docs/guides/tutorials/getting_started.rst

```diff
@@ -4,9 +4,9 @@
 Getting started with Great Expectations
 ###############
 
-Welcome to Great Expectations! This tutorial will help you set up your first local deployment of Great Expectactions. We'll also introduce important concepts, with links to detailed material you can dig into later.
+Welcome to Great Expectations! This tutorial will help you set up your first local deployment of Great Expectations that contains a small Expectation Suite to validate some sample data. We'll also introduce important concepts, with links to detailed material you can dig into later.
 
-Please follow these steps to get started:
+The tutorial will walk you throw the following steps - **scroll to the bottom of this page and click the "Next" button to jump to the start of the tutorial!**
```

Comment:
I removed the individual links from each bullet here and just kept the explanations as an overview of the tutorial content, since I found it really confusing to have links here, instead of just following the "next" button.

Reply:
I agree. In fact, I think we should remove the huge numbers and font and make the overview here much slimmer, since you're going to recommend scrolling past it.

Reply:
Removing the links makes sense to me. I still like the visual hierarchy of the numbers as a way to make the flow clear.